### PR TITLE
feat: add types of multiselect

### PR DESCRIPTION
**Pull request by nagamejun**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/multiselect/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// Type definitions for multiselect 0.9
+// Project: https://github.com/lou/multi-select
+// Definitions by: nagamejun <https://github.com/me>
```
Comment (danger-public) [inline types/multiselect/index.d.ts:3]: - :no_entry_sign: This line should have your github username in it, not /me

Comment (sheetalkamat) [inline types/multiselect/index.d.ts:3]: Fix user name

File: types/multiselect/multiselect-tests.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { Options, AddOption } from 'multiselect';
```
Comment (sheetalkamat) [inline types/multiselect/multiselect-tests.ts:1]: Add test for global use as well as module (do not use import statement in global test) 
https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/multiselect/multiselect-tests.ts (hunk 2 above)
Comment (nagamejun) [reply types/multiselect/multiselect-tests.ts]: Thank you for the review.
Removed unnecessary declare global.

File: types/multiselect/index.d.ts (hunk 1 above)
Comment (peterblazejewicz) [inline types/multiselect/index.d.ts]:
```suggestion
// Definitions by: nagamejun <https://github.com/nagamejun>
```

Comment (nagamejun) [reply types/multiselect/index.d.ts]: I'm done. 971aae3

**Review by amcasey — APPROVED** [review]

**Comment by nagamejun** [comment]
Ready to merge

**Pull request merged**